app/views/selfupdate_beta.scala.txt

```diff
@@ -140,8 +140,8 @@ if [[ -z $(cat ${sdkman_config_file} | grep 'sdkman_debug_mode') ]]; then
 	echo "sdkman_debug_mode=true" >> "$sdkman_config_file"
 fi
 
-if [[ -z $(cat ${sdkman_config_file} | grep 'sdkman_colour_enable') ]]; then
-	echo "sdkman_colour_enable=true" >> "$sdkman_config_file"
+if [[ -z $(cat ${sdkman_config_file} | grep 'sdkman_colour_enable|sdkman_auto_update') ]]; then
```

Comment:
Not sure but was this intentional?

It would be great if we could migrate the `sdkman_selfupdate_enable` config to `sdkman_auto_update`. We wouldn't want people to lose their config because we changed the name.

Reply:
This was a definitely not intentional 😄 . So if someone has set `sdkman_selfupdate_enable=false`, we should replace it with `sdkman_auto_update=false`?

Reply:
> So if someone has set `sdkman_selfupdate_enable=false`, we should replace it with `sdkman_auto_update=false`?

Yes, I think so, it will be smoother for the users who depend on this setting already.